### PR TITLE
[EZ] GET/ Subscription_v1 HMAC Key Change

### PR DESCRIPTION
**Pull request by amarjandu**
replaces `hmac_secret_key` with `hmac_key_id` for get/find subscription operations.

### CODE REVIEW

File: dss/api/subscriptions_v1.py [hunk 1]
```diff
@@ -35,8 +35,8 @@ def get(uuid: str, replica: str):
     source = response['_source']
     source['uuid'] = uuid
     source['replica'] = replica
-    if 'hmac_secret_key' in response.keys():
-        source['hmac_secret_key'] = response['hmac_secret_key']
+    if 'hmac_key_id' in response.keys():
```
Comment (hannes-ucsc) [inline dss/api/subscriptions_v1.py]:
```suggestion
    if 'hmac_key_id' in response:
```

File: dss/api/subscriptions_v1.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def find(replica: str):
         'uuid': hit.meta.id,
         'replica': replica,
         'owner': owner,
-        **{k: v for k, v in hit.to_dict().items() if not k.startswith('hmac_key_id')}}
+        **{k: v for k, v in hit.to_dict().items() if not k.startswith('hmac_secret_key')}}
```
Comment (hannes-ucsc) [inline dss/api/subscriptions_v1.py]:
```suggestion
        **{k: v for k, v in hit.to_dict().items() if k != 'hmac_secret_key'}}
```

**Comment by kislyuk** [comment]
Shouldn't the Swagger YAML be updated as well?

File: dss-api.yml [hunk 3]
```diff
@@ -2295,9 +2295,9 @@ definitions:
           number of attempts is reached.
         type: string
         pattern: 'https?://'
-      hmac_secret_key:
+      hmac_key_id:
         description: >
-          Key used to verify authenticity of subscription
+          id used to verify authenticity of subscription
```
Comment (xbrianh) [inline dss-api.yml]: This should be changed to "The HMAC key-id"

**Pull request merged**